build_deploy.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -ex
+
+QUAY="quay.io/app-sre"
+VERSION=`grep 'ENV SENTRY_VERSION' Dockerfile | cut -d' ' -f 3 | tr -d '\r\n'`
+TARGET="${QUAY}/sentry:${VERSION}"
+
+docker build --pull -t ${TARGET} .
+docker push ${TARGET}
+
+exit 0
```

Comment:
not needed?

Reply:
being explicit doesn't hurt! I don't mind the `exit 0`